flocker/node/_change.py

```diff
@@ -27,6 +29,11 @@ class IStateChange(Interface):
     """
     An operation that changes local state.
     """
+    eliot_action = Attribute(
```

Comment:
Oh.  Uh.  That's not how I thought Eliot worked.

Now I see that `ActionType.__call__` does a `startAction` before returning.  So:

```
X = ActionType(...)
y = X(...)
```

Actually starts the action on the second line.  This basically invalidates everything I said above about getting rid of properties.  Also... blech?  I think?  This seems like a surprising way for it to work.

So now I agree that this should be a method, not a property.  But I still don't feel like fixing it.  Maybe I will in the morning... except then I'll have to get in the three hour build queue... So in the interest of not wasting that time tomorrow, I'm going to merge this as-is (which is what I was going to do anyway, except now I know it's wrong instead of thinking maybe it's right).  Someone can convert it to a method later.